### DISCUSSION TITLE
Sort the Estimation Matrix from Newest to Oldest

### DISCUSSION THREAD

**Discussion by Fraser35** [Zuehlke/poinz#369] [Ideas]
As a team, we want to compare previous estimations when sizing something new, but generally more recent estimations will be more relevant than older ones. It would be great to have the default sorting from Newest to Oldest, or even better to be able to select what sorting to be used (a bit like the sorting available on the backlog, but probably with different options)

**Comment by xeronimus** [comment]
Thanks for your input.

I'd suggest a "sort" button with dropdown (similar to the backlog) with 4 options
- Newest
- Oldest
- Title A-Z
- Title Z-A

I would place this sort button in the top right corner of the Estimation Matrix, next to "Include Trashed stories"
